src/transform/transform.ts

```diff
@@ -223,7 +223,19 @@ function processOperationRequests(session: Session<CodeModel>) {
   for (const group of values(session.model.operationGroups)) {
     for (const op of values(group.operations)) {
       if (op.requests!.length > 1) {
-        throw console.error('multiple requests NYI');
+        for (const req of values(op.requests)) {
+          const newOp = JSON.parse(JSON.stringify(op));
+          newOp.requests = (<Array<Request>>op.requests).filter(r => r === req);
+          let name = op.language.go!.name;
+          if (req.protocol.http!.knownMediaType === 'json') {
+            name = name + 'With' + req.parameters![0].schema.language.go!.name;
+          }
+          newOp.language.go!.name = name;
+          newOp.language.go!.protocolNaming = new protocolMethods(newOp.language.go!.name);
+          group.addOperation(newOp);
+        }
+        group.operations.splice(group.operations.indexOf(op), 1);
+        continue;
```

Comment:
Looks like this `continue` will skip the rest of the processing for this operation.  I wonder if we need to perform the operation split first, in a separate loop?

Reply:
I think performing the continue here is fine because if you see right above I am removing the original operation since it it unnecessary from this point onwards and wont ever be used. Per request, the operation is copied and appended to the list of operations. Each of those new operations will be processed fully.

Reply:
D'oh I missed that, makes sense.